spec/ddtrace/profiling/integration_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+require 'ddtrace'
+require 'ddtrace/profiling/pprof/builder'
+require 'ddtrace/profiling/collectors/stack'
+require 'ddtrace/profiling/recorder'
+require 'ddtrace/profiling/scheduler'
+require 'ddtrace/profiling/exporter'
+require 'ddtrace/profiling/encoding/profile'
+
+RSpec.describe 'profiling integration test' do
```

Comment:
Basic sanity test for profiling that runs it end-to-end synchronously. We can embellish on this as profiling continues to develop.